src/helpers/presentation.rs

```diff
@@ -108,6 +116,131 @@ impl<'a> Presentation<'a> {
             .map_err(|err| PresentationError::VadeEvanError(err.to_string()))
     }
 
+    /// Creates a presentation.
+    /// The presentation has proof and requested credentials.
+    ///
+    /// # Arguments
+    ///
+    /// * `proof_request` - proof request for presentation
+    /// * `credential` - credential to be shared in presentation
+    /// * `master_secret` - user's master secret
+    /// * `signing_key` - users secp256k1 private signing key
+    /// * `revealed_attributes` - list of names of revealed attributes in specified schema,
+    ///
+    /// # Returns
+    /// * `Option<String>` - A `Presentation` as JSON
+    pub async fn create_presentation(
```

Comment:
Can you check the usage of `PresentationError::VadeEvanError`?

This error type is only intended to be used when a call against an internal `Vade` instance failed. For the other errors handled here, we could add a separate error for invalid input, etc. ^^

Reply:
done, updated the code to use VadeEvanError only for vade calls.